pkg/gui/filetree/file_node.go

```diff
@@ -30,6 +30,17 @@ func (self *FileNode) GetHasUnstagedChanges() bool {
 	return self.SomeFile(func(file *models.File) bool { return file.HasUnstagedChanges })
 }
 
+func (self *FileNode) GetStagedFiles() []*models.File {
+	stagedFiles := make([]*models.File, 0)
+	self.ForEachFile(func(t *models.File) error {
```

Comment:
@stefanhaller I'm not sure if there's a better way of doing this, if not distinguishing `ForEachFileErr` from `ForEachFile` (that accepts a function that doesn't return an error). Any suggestions?

Reply:
Since we know that the lambda expression will never return an error, we can simply silence the compiler warning with `_ = self.ForEachFile( ...`.

However, I think you don't need this function at all, so the issue is moot.